### ISSUE TITLE
Autogenerate password as user types (remove Generate button)

### ISSUE THREAD

**Issue by autonome** [lesspass/lesspass#505]
The notes here describe an intent to eventually remove the Generate button:

https://github.com/lesspass/lesspass/issues/123#issuecomment-273791173

**Comment by guillaumevincent** [comment]
Yes, we can debounce and generate the master password automatically.
We can also at the end of the process, copy it directly: See https://github.com/lesspass/lesspass/blob/master/sandbox/copy-paste-async.html

**Comment by guillaumevincent** [comment]
The only issue for now is the Firefox copy paste async not working.
We need to make the feature backward compatible.
